bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/i18n/nikohomecontrol.properties

```diff
@@ -112,15 +158,10 @@ offline.configuration-error.ip = Cannot resolve bridge IP with given host name
 offline.configuration-error.tokenEmpty = Hobby API token is empty
 offline.configuration-error.tokenExpired = Hobby API token has expired
 
-offline.configuration-error.actionId = Configured action ID does not match an action in controller
-offline.configuration-error.actionType = Unsupported action type
-offline.configuration-error.actionRemoved = Action has been removed from controller
+offline.configuration-error.deviceId = Configured device ID does not match a device in controller
+offline.configuration-error.deviceRemoved = Device has been removed from controller
 
-offline.configuration-error.energyMeterId = Configured energy meter ID does not match an energy meter in controller
-offline.configuration-error.energyMeterRemoved = Energy meter has been removed from controller
-
-offline.configuration-error.thermostatId = Configured thermostat ID does not match a thermostat in controller
```

Comment:
This entry is still used in class ThermostantHandler

Reply:
Changed